File: Angular-RecipeBook-BackEnd/Application.BusinessLogicLayer/Modules/RecipeModule/ResponseModels/EditRecipeResponseModel.cs
using System.Collections.Generic;

namespace Application.BusinessLogicLayer.Modules.RecipeModule.ResponseModels
{
    public class EditRecipeResponseModel
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string ImagePath { get; set; }

        public List<EditRecipeIngredientListItemResponseModel> Ingredients { get; set; }
    }
}

File: Angular-RecipeBook-FrontEnd/src/app/recipes/models/response-models/edit-recipe-response.model.ts
import { EditRecipeIngredientListItemResponseModel } from './edit-recipe-ingredient-list-item-response.model';

export interface EditRecipeResponseModel {
  id: number;
  name: string;
  description: string;
  imagePath: string;
  ingredients: EditRecipeIngredientListItemResponseModel[];
}
